pages/api/map.tsx

```diff
@@ -1,12 +1,14 @@
 import type { NextApiRequest, NextApiResponse } from 'next'
+
 import kommuner from '../data/kommuner.json'
+import { ONE_WEEK_MS } from '../../utils/shared'
 
 export default function userHandler(req: NextApiRequest, res: NextApiResponse) {
   const { method } = req
 
   switch (method) {
     case 'GET':
-      res.setHeader('Cache-Control', 'public, s-maxage=604800') // a week
```

Comment:
`s-maxage` is an invalid cache header. This was a bug that prevented caching completely.